sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -145,9 +145,17 @@ object StaticSQLConf {
         "cause longer waiting for other broadcasting. Also, increasing parallelism may " +
         "cause memory problem.")
       .intConf
-      .checkValue(thres => thres > 0 && thres <= 128, "The threshold must be in [0,128].")
+      .checkValue(thres => thres > 0 && thres <= 128, "The threshold must be in (0,128].")
       .createWithDefault(128)
 
+  val SUBQUERY_MAX_THREAD_THRESHOLD =
```

Comment:
I think we should submit a separate PR for this change.

Reply:
I think we could keep this in StaticSQLConf.

- I locally test the static config takes effect, it can only be set while startup.
- The UT can pass because it was used in lazy val `SubqueryExec.relationFuture` on the executor side, so the `withSQLConf` in UT could set the config before executor start.

cc @hvanhovell